internal/clients/http/sources/sources_client.go

```diff
@@ -274,6 +286,40 @@ func (c *sourcesClient) GetProvisioningTypeId(ctx context.Context) (string, erro
 	return appTypeId, nil
 }
 
+func (c *sourcesClient) GetSourceTypes(ctx context.Context) (map[ID]string, error) {
+	sourceTypes, err := c.loadSourceTypes(ctx)
```

Comment:
I guess that if we don't find a better solution, we can cache this whole map[ID]string :shrug: 

@lzap opinion whether its a good idea? :thinking:

Reply:
Hmmm how about refactoring [global type id](https://github.com/RHEnVision/provisioning-backend/blob/main/internal/cache/redis.go#L36) we hold in cache package as a global variable into struct type `AppConstants` or something like that having ID as one field and this map as another.

All you need is to implement the Cacheable interface and register the type as a GOB serializeable type, all [in the same file](https://github.com/RHEnVision/provisioning-backend/blob/main/internal/cache/redis.go#L36).